Cargo.toml

```diff
@@ -30,3 +30,8 @@ tokio = { version = "1", features = ["time", "rt"] }
 default = ["unicode-width", "console/unicode-width"]
 improved_unicode = ["unicode-segmentation", "unicode-width", "console/unicode-width"]
 in_memory = ["vt100"]
+
+[[test]]
+name = "render"
+path = "tests/render.rs"
+required-features = ["in_memory"]
```

Comment:
Instead of this, I'd prefer putting a `#![cfg(feature = "in_memory")]` at the top of `render.rs`.